src/Controller/ActivityController.php

```diff
@@ -470,6 +474,24 @@ public function createAction(
 
         $activityRepository->save($newActivity);
 
+        $message = (new Swift_Message(
+            $owner->getName() . ' ' . $owner->getSurname() .
+            ' has created a new activity waiting for your validation '
+        ))
+            ->setFrom($_ENV['EMAIL_FROM'])
+            ->setTo($owner->getProjectManager()->getEmail())
+            ->setBody(
+                $this->renderView(
+                    'mail/mail_for_PM.html.twig',
```

Comment:
`mail_for_PM` doesn't indicate what mail is this. Is it that a user has been assigned to the PM, is it a new job to validate e.t.c.